src/pydata_sphinx_theme/__init__.py

```diff
@@ -1166,6 +1166,9 @@ def copy_logo_images(app: Sphinx, exception=None) -> None:
         path_image = logo.get(f"image_{kind}")
         if not path_image or isurl(path_image):
             continue
+        if os.path.isfile(staticdir / Path(path_image).name):
```

Comment:
```suggestion
        if (staticdir / Path(path_image).name).is_file():
```
I would stick to pathlib instead of `os.path`

Reply:
Fixed!